src/commands/server/deploy.ts

```diff
@@ -69,8 +69,9 @@ export default class Deploy extends Command {
     [LOG_DIRECTORY_KEY]: logsDirectory,
     multiuser: flags.boolean({
       char: 'm',
-      description: 'Starts Eclipse Che in multi-user mode',
+      description: 'Deprecated. The flag is ignored. Eclipse Che is deployed in multi-user mode by default.',
```

Comment:
To  me `Eclipse Che is always deployed in multi-user mode` is more clear, because if you are talking about defaults, then I have feeling that I can change the defaults, which is not true in this case.